budget-app/src/components/organisms/HomePage/HomePage.tsx

```diff
@@ -46,7 +46,7 @@ const HomePage: React.FC = () => {
         <Route path="/Charts" component={Charts} />
         <Route path="/Cards" component={Cards} />
       </Switch>
-      <UserInfro transactions={transactions} />
+      <UserInfro />
```

Comment:
info * 

but i think you should change the name of the comp because you got more then just userInfo inside of it

Reply:
what this guy said